protos/flyteidl/service/dataproxy.proto

```diff
@@ -122,16 +122,21 @@ message GetDataRequest {
   // backend, identifies a Flyte artifact ([i]nput, [o]utput, flyte [d]eck, etc.).
   // e.g. flyte://v1/proj/development/execid/n2/0/i (for 0th task execution attempt input)
   //      flyte://v1/proj/development/execid/n2/i (for node execution input)
+  //      flyte://v1/proj/development/execid/n2/o/o3 (the o3 output of the second node)
   string flyte_url = 1;
 }
 
 message GetDataResponse {
   oneof data {
     // literal map data will be returned
-    core.LiteralMap literal_map                = 1;
+    core.LiteralMap literal_map = 1;
 
     // Flyte deck html will be returned as a signed url users can download
     PreSignedURLs pre_signed_urls = 2;
+
+    // Single literal will be returned. This is returned when the user/url requests a specific output or input
+    // by name. See the o3 example above.
+    core.Literal literal = 3;
```

Comment:
since either `literal_map` or `literal` are set at any point in time, does it make sense to move this to a `oneof`?